LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Buildium, LLC

```

Comment:
Some of our FE packages copyright as **Buildium**, but it seemed more accurate to use **Buildium, LLC** as our official name.